includes/Modules/Analytics_4.php

```diff
@@ -2177,9 +2160,10 @@ private function filter_analytics_allow_tracking_disabled( $allowed ) {
 	private function set_available_audiences( $audiences ) {
 		$available_audiences = array_map(
 			function( GoogleAnalyticsAdminV1alphaAudience $audience ) {
+				$display_name  = $audience->getDisplayName();
 				$audience_item = array(
 					'name'        => $audience->getName(),
-					'displayName' => $audience->getDisplayName(),
+					'displayName' => ( 'All Users' === $display_name ) ? __( 'All Visitors', 'google-site-kit' ) : $display_name,
```

Comment:
Please note the value for "All visitors" should have a lower-case initialled "visitors", and it should also _not_ be translated (the value it's replacing is not translated and is always created in English):

```suggestion
					'displayName' => ( 'All Users' === $display_name ) ? 'All visitors' : $display_name,
```